kubemarine/keepalived.py

```diff
@@ -276,6 +277,6 @@ def configure(group: NodeGroup) -> RunnersGroupResult:
 
     log.debug(group.sudo('ls -la %s' % package_associations['config_location']))
 
-    restart(group)
+    enable(group)
```

Comment:
It should be possible to change keepalived configuration in `cluster.yaml` and run `install --tasks prepare.loadbalancer.keepalived.configure` to configure and apply the changes. I think restart of keepalived should be kept.

Reply:
If you mean `install --tasks deploy.loadbalancer.keepalived.configure`, then you are right. I did both restart and enable. This will not break anything and will allow the unit to start correctly.